src/monster.cpp

```diff
@@ -458,12 +449,54 @@ int monster::print_info(WINDOW* w, int vStart, int vLines, int column) const
 
 std::string monster::extended_description() const
 {
-    // @todo Add tons of info here
-    std::string ret = string_format( _ ( "This is a %s" ), name().c_str() );
-    ret += "\n--\n";
+    std::ostringstream ss;
+
+    nc_color color = c_white;
+    std::string attitude = "";
+    get_Attitude( color, attitude );
+    attitude = get_tag_from_color( color ) + attitude;
+
+    ss << string_format(
+           _( "This is a %s %s <stat>%s</stat>" ),
```

Comment:
You sure this will translate well? I mean the attitude part - is it in proper case and all that?
`game.cpp` uses it without surrounding text, meaning it might not be suitable for usage in a full sentence.